src/pages/Profile.css

```diff
@@ -0,0 +1,66 @@
+.formContainer {
+    border: solid #c49e34;
+    border-radius: 5%;
+    width: 50%;
+    padding: 0.8rem;
+    margin: auto;
+    margin-top: 2%;
+    background-color: #e7e7e7;
+    z-index: 1;
+}
+
+.bottleForm {
+    display: grid;
+    grid-template-columns: repeat(2, 0.1fr);
+    text-align: center;
+    justify-content: center;
+    margin-top: 3rem;
+}
+
+.inputBottle {
+    font-weight: bold;
+    font-size:1.5rem;
+    padding: 0.25rem;
+    border: 1px solid black;
+    background-color: white;
+    border-radius: 5px;
+    margin: 10px;
```

Comment:
you should use rem for margins, and no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

Reply:
Fixed